src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.spec.ts

```diff
@@ -315,6 +319,26 @@ describe('DotEditLayoutComponent - Template (anonymous = false)', () => {
         expect(dialog.styles.display).toEqual('block');
     });
 
+    it('should set edit template mode', () => {
+        fixture.detectChanges();
+        spyOn(component, 'setEditLayoutMode');
```

Comment:
I dont know if we should mock this method because it is a method of the component that we are testing